plugins/catalog-backend/api-report.md

```diff
@@ -389,6 +395,11 @@ export const locationSpecToLocationEntity: typeof locationSpecToLocationEntity_2
 // @public @deprecated (undocumented)
 export const locationSpecToMetadataName: typeof locationSpecToMetadataName_2;
 
+// @public (undocumented)
+export function parseEntityOrderFieldParams(
```

Comment:
Let's skip this and just copy it for now. We don't want the notifications-backend to depend on catalog-backend anyway. If this becomes a frequent thing we can look for an opportunity to provide some common utilities as part of the framework instead